DSA-Campaign-Uplift-Estimation/src/main/webapp/Create/create-script.js

```diff
@@ -20,8 +20,20 @@ let keywordCampaignId = null;
 // number of locations
 let locationCount = 1;
 
+// number of negative locations
+let negLocationCount = 1;
+
+// preset index selected.
+let selectedPreset = -1;
+
 // array to keep track of all preset names belonging to user
-const userPresets = [];
+const USERPRESETS = [];
+
+// length of mm-dd-yyyy (error handling)
+const DATE_LENGTH = 10;
+
+// number of max presets (to prevent random page behavior)
+const MAX_PRESETS = 20;
```

Comment:
Why would there be random page behavior?

Reply:
When there are too many presets, it extends the page beyond the form and makes it unorganized

Reply:
Changed comment to be more clear.